Readme.md

```diff
@@ -43,6 +43,13 @@ When you are ready, you can deploy smart contracts using truffle migrations.
 - `truffle migrate --network development` to deploy to local ganache test blockchain
 - `truffle migrate --network ropsten` to deploy to ethereum testnet
 
+## Commands
+We strongly recommend you to run `npx`.
```

Comment:
```suggestion
You can use `npx` for running command line tools bundled with the project.
```
I don't think we should "strongly recommend" as it's really up to the contributor to define their workflow.